airsonic-main/src/main/java/org/airsonic/player/controller/GeneralSettingsController.java

```diff
@@ -77,6 +81,9 @@ protected void formBackingObject(Model model) throws Exception {
             }
         }
 
+        String currentAdminUser = settingsService.getAdminUser();
```

Comment:
This variable is unused.

Reply:
Ah yeah, I refactored out the part that used that